File: Python/1035.py
values = input().split(" ")
a, b, c, d = values
a, b, c, d = int(a), int(b), int(c), int(d)


if c > 0 and d > 0 and a % 2 == 0:
    if b > c and d > a:
        if (c + d) > (a + b):
            print("Valores aceitos")
        else:
            print("Valores nao aceitos")
    else:
        print("Valores nao aceitos")
else:
    print("Valores nao aceitos")

File: Dart/1035.dart
import 'dart:io';

void main() {
  String inputs = stdin.readLineSync();
  List inputsList = inputs.split(" ");
  inputsList = inputsList.map((e) => int.parse(e)).toList();
  int a, b, c, d;
  a = inputsList[0];
  b = inputsList[1];
  c = inputsList[2];
  d = inputsList[3];
  if (b > c && d > a && (c + d) > (a + b) && c > 0 && d > 0 && a % 2 == 0) {
    print("Valores aceitos");
  } else {
    print("Valores nao aceitos");
  }
}
